modules/Microsoft.Cache/redis/.test/full.parameters.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Should this be called parameter.json?

Reply:
It's very inconsistent through all modules. As long as the reference in the Azure DevOps pipeline definition is correct this should be fine.
Can't wait to switch to bicep based parameter files better sooner than later 😉

Reply:
Hey @itpropro, I'm with you on bicep test files 😄 However, consider a test file naming pattern will still be needed once moving to .bicep tests, since the plan is to still use different files for different use cases. 
You're right about inconsistency, that's why we opened an issue to discuss that (see issue #1564). For the moment, I'd suggest to rename the most complete parameter file as `parameters.json` which is the most commonly used throughout the library.

Reply:
@itpropro I agree, it's a bit of a mixed bag. Personally, I don't really care for the name of the file as each just represents a test case and as such has the sole pupose of evaluation certain aspects of the module. In case of, for example, CosmosDB we have 4 files for the 4 different data base types - with no `parameters.json` in sight. 
I can't however also argue with the fact that the most common name we have today is `parameters.json` if the parameter file is just a general one without the expectation to test all, or e.g. the minimal amount of parameters. (Sidenote: To add to the inconsistency, some modules also have a `max.parameters.json` :D )